local/bin/py/update_pre_build.py

```diff
@@ -331,6 +333,40 @@ def process_integration_metric(self, file_name):
             new_file_name = '{}{}.yaml'.format(self.data_integrations_dir, key_name)
             self.csv_to_yaml(key_name, file_name, new_file_name)
 
+    def dev_doc_integrations_core(self, file_name):
+        """
+        Take the content from https://github.com/DataDog/integrations-core/tree/master/docs/dev 
+        and transform it to be displayed on the doc in the /developers/integrations section
+        :param file_name: path to a file
+        """
+        relative_path_on_github = '/integrations-core/docs/dev/'
+        doc_directory = '/developers/integrations/'
+
+        if (relative_path_on_github in file_name and file_name.endswith('.md')): 
+            
+            # Replacing the master README.md by _index.md to follow Hugo logic
+            
+            if file_name.endswith('README.md'):
+                doc_file_name = '_index.md'
+            else:
+                doc_file_name = basename(normpath(file_name))
+            
+            with open(file_name, mode='r+') as f:
+                content = f.read()
+                
+                # Replacing H1 title from Github with the front_matter style from the doc
+                content = re.sub(self.regex_h1_replace, r'---\ntitle:\2\ntype: documentation\n---\n', content, count=0)
```

Comment:
I think we can remove this by implementing the doc front_matter logic in the integrations core repo files directly. It won't break the rendering of md files in the github UI, and will allow us to add descriptions and other param if need be.